### PR TITLE
update lessons

### PR DESCRIPTION
**Pull request by icorson3**
@AliSchlereth added more detail to intro to html and css lessons. The goal is for them to build their own little personal page and make it fun. I don't want to give them too many constraints. But let me know what you think.

### CODE REVIEW

**Comment by icorson3** [comment]
@AliSchlereth I realized that I added the advanced CSS stuff here as well. meant to checkout a branch but forgot 🤦‍♀️. Let me know what you think

**Comment by AliSchlereth** [comment]
@icorson3  This is great information, thank you for your additions, they're super helpful. 

It might be good to add vocab, warm up questions and recap questions to the intro to css lesson. Would you prefer to do it or should I add those in when I review/prep the lesson?

**Comment by icorson3** [comment]
@AliSchlereth looking at frontend lessons makes me want to redo everything! LOL...I think we could leave out the reference to that lesson but I like the idea of introducing basic web tools, how to open them and inspect. Do you want to add that and the vocab etc sections? Thanks for taking a look!

File: module2/lessons/intro_to_html.md [hunk 1]
```diff
@@ -171,22 +173,23 @@ title: Introduction to HTML
   ```
   - A lot of the tags we use are already semantic such as form, link, etc.
 
-### Block vs. Inline
-
-  - All HTML tags have a default display value. Either block or inline.
+### Display Properties: Block vs. Inline
+  - The default value for all HTML tags is `inline`. Most "User Agent stylesheets" (the default styles the browser applies) reset many elements to `block`
```
Comment (icorson3) [inline module2/lessons/intro_to_html.md]: I didn't know this! Interesting!

**Comment by icorson3** [comment]
@AliSchlereth looks good to me, I am going to merge!

**Pull request merged**